samples/custom_file/enclave.json

```
@@ -0,0 +1,14 @@
+{
+ "exe": "hellofile",
+ "key": "private.pem",
+ "debug": true,
+ "heapSize": 512,
+ "productID": 1,
+ "securityVersion": 1,
+ "mounts": null,
+ "env": null,
+ "file": [{
+     "source": "/etc/ssl/certs/ca-certificates.crt",
+     "target": "/ca-certificates.crt"
```

```suggestion
     "target": "/etc/ssl/certs/ca-certificates.crt"
```
Can we place this at the same location so that Go finds the certificates without explicitly loading them?

---

Directories will now be generated automatically, so this works now : )